src/apify_helpers.js

```diff
@@ -213,6 +213,168 @@ const getPrefilledValuesFromInputSchema = (inputSchemaStringJSON) => {
     return prefilledObject;
 };
 
+/**
+ * Converts Apify input schema to Zapier input fields.
+ * Input schema spec.
+ * https://docs.apify.com/platform/actors/development/actor-definition/input-schema Fields schema
+ * spec. https://zapier.github.io/zapier-platform-schema/build/schema.html#fieldschema
+ * @param inputSchema
+ * @param actor
+ */
+const createFieldsFromInputSchemaV1 = (inputSchema, actor) => {
+    const { properties, required, description } = inputSchema;
+    const fields = [
+        // The first fies is info box with input schema description or actor title, same as on Apify platform.
+        {
+            label: actor.title,
+            key: `actor-${actor.id}-info`,
+            type: 'copy',
+            helpText: description || `${actor.title} Input, see [documentation](https://apify.com/${actor.username}/${actor.name}) `
+                + 'for detailed fields description.',
+        },
+    ];
+    // eslint-disable-next-line no-restricted-syntax
+    for (const [propertyKey, definition] of Object.entries(properties)) {
+        // eslint-disable-next-line no-continue
+        if (definition.editor === 'hidden') continue;
+        // NOTE: Handle sectionCaption with info box with helpText. It is not possible to do stackable fields in Zapier.
+        if (definition.sectionCaption && definition.sectionCaption.length) {
+            const helpText = definition.sectionDescription
+                ? `${definition.sectionCaption} - ${definition.sectionDescription}`
+                : definition.sectionCaption;
+            fields.push({
+                label: definition.sectionCaption,
+                key: `sectionCaption-${propertyKey}`,
+                type: 'copy',
+                helpText,
+            });
+        }
+        const field = {
+            label: definition.title,
+            helpText: definition.description,
+            key: propertyKey,
```

Comment:
Maybe we should prefix propertyKey with something - adding eg. `actorId` to input (which is possible) would interfere with the `actorId` field that is added automatically, wouldn't it?